server/server_linux.go

```diff
@@ -12,12 +12,6 @@ import (
 
 // apply sets config settings on the server process
 func apply(ctx context.Context, config *Config) error {
-	if !config.NoSubreaper {
```

Comment:
For backwards compatibility, I think we need to leave this in as deprecated then warn about its usage.

Reply:
It won't fail if set and is only used internally.  Not any external functions to this option.